### ISSUE TITLE
[10.7.0-rc3] Docker - Anime Plugin causes a boot loop in docker

### ISSUE THREAD

**Issue by SkalCore** [jellyfin/jellyfin#5268]
**Describe the bug**

**System (please complete the following information):**
 - OS: [Linux]
 - Virtualization: [Docker]
 - Clients: [Browser]
 - Browser: [Firefox 85.0.2]
 - Jellyfin Version: [10.7.0-rc3]
 - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
 - Installed Plugins: [none, because Anime Plugin is causing a bootloop.]
 - Reverse Proxy: [traefik.]
 - Base URL: [none]

**To Reproduce**

1. install plugin
2.  restart docker container
3. bootloop occurs
4. delete Anime Plugin from plugin folder
5. Docker container starts normal

**Comment by crobibero** [comment]
https://github.com/jellyfin/jellyfin-plugin-anime/issues/94#issuecomment-744870138

tldr The anime plugin has split.
